repo/packages/C/calico/0/config.json

```diff
@@ -0,0 +1,169 @@
+{
+  "properties": {
+    "calico": {
+      "description": "Calico specific configuration properties.",
+      "properties": {
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "calico"
+        },
+        "calicoctl-url": {
+          "type": "string",
+          "description": "The download URL for the calicoctl command line tool.",
+          "default": "{{resource.assets.uris.calicoctl}}"
```

Comment:
this is incorrect and is causing a failure on installation... the calioctrl-url property in this file should not exist at all.